### ISSUE TITLE
Having only one region in toml caused issue 

### ISSUE THREAD

**Issue by jennieli421** [LooseLab/readfish#353]
I wanted to perform a run with only enrichment, so I had one region specified in TOML like the following. But I did not see any unblock actions from the live state panel. The purple peak indicating voltage reversals did not appear at all in the histogram. However, everything works fine as long as I add a region with `control=true`. I also tried with two regions with `control=false` and that also didn't have any rejection. 

```
[caller_settings.guppy]
config = "dna_r10.4.1_e8.2_400bps_5khz_fast_prom.cfg" # PromethION
address = "ipc:///tmp/.guppy/5555"

[mapper_settings.mappy_rs]
fn_idx_in = "comparison.mmi"
n_threads = 8

[[regions]]
name = "analysis"
control = false
min_chunks = 1 # minimum number of chunks before a decision can be made
max_chunks = 1 # maximum number of chunks to use in decision making - after this perform the above_max_chunks action
targets = []  # Genomic targets for this region
single_on = "unblock"
multi_on = "unblock"
single_off = "stop_receiving"
multi_off = "stop_receiving"
no_map = "proceed"
no_seq = "proceed"
above_max_chunks = "unblock"  # Action to take if the number of chunks received is above max_chunks
below_min_chunks = "proceed"  # Action to take if the number of chunks received is below min_chunks
```

**Comment by Adoni5** [comment]
`max_chunks` has to be greater than `min_chunks`, and should not be set to **1.**
You also don't seem to have set any targets?

**Comment by jennieli421** [comment]
We have been using `max_chunks=1` for a very long time and the results were fine. Here I am only providing targeted region's reference, so I don't need to specify `targets`

**Comment by mattloose** [comment]
> Here I am only providing targeted region's reference, so I don't need to specify targets

I'm not quite sure what you are trying to do here. If you do not give any targets then you will not get any unblocking activity. An empty 
>targets = []
will mean that no targets are checked and so the later logic of on or off target will not be evaluated.

An example toml enriching for two chromosomes from a reference is available here:
https://github.com/LooseLab/readfish/blob/main/docs/_static/example_tomls/human_chr_selection.toml

In short, you must have something in the targets = [] for any unblocking activity to occur.

**Comment by jennieli421** [comment]
I have been using this setting (with split region) for a long time and it is working as expected. The design for not having targets can be traced back to this conversation: 
https://github.com/LooseLab/readfish/issues/242#issuecomment-1610971660

Basically I have provided references of only the desired sequences, so everything mapped to the reference is accepted.

**Comment by mattloose** [comment]
Hmmm

I indeed stand corrected.

Let me check this further and come back to you.

**Comment by mattloose** [comment]
Hi, Could you pleaase confrim your readfish version and also share your readfish log with us - the log should be called the <run_id>.tsv

Thanks

Matt

**Comment by jennieli421** [comment]
[tested_one_or_more_regions_ef170d0f.zip](https://github.com/LooseLab/readfish/files/15251171/tested_one_or_more_regions_ef170d0f.zip)

Unfortunately, I was testing different region settings during the run, so this run had multiple settings. I hope this is fine.

**Comment by Adoni5** [comment]
Hi @jennieli421 - do these logs contain an example of it working, with a region of control==True, where you see unblocking?

And can I just check the version of `readfish` you were using?

```bash
readfish --version
```

**Comment by jennieli421** [comment]
```
readfish --version
readfish 2023.1.1
```
Both should contain multiple trials where I see unblocking, as long as there is a region of `control=True`. 

![Screenshot 2024-05-12 at 1 03 25 AM](https://github.com/LooseLab/readfish/assets/69633721/ec65d733-ec1b-4ed3-bd50-855a7931dd09)
![Screenshot 2024-05-12 at 1 03 42 AM](https://github.com/LooseLab/readfish/assets/69633721/fc7735fd-a79a-4c92-b766-b17936960292)

**Comment by jennieli421** [comment]
Hello, Are there any updates regarding this issue?
